kernel-headers/rdma/efa-abi.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-2-Clause) */
 /*
- * Copyright 2018-2022 Amazon.com, Inc. or its affiliates. All rights reserved.
+ * Copyright 2018-2023 Amazon.com, Inc. or its affiliates. All rights reserved.
```

Comment:
Same as below, the latest open source policy advises dateless notices, e.g.
```
Copyright Amazon.com, Inc. or its affiliates. All rights reserved.
```

Reply:
@wenduwan Thanks.
This is a kernel header, can't change it here so will leave dated copyrights for now.